### PR TITLE
Typos + Spark-Shell Update

### PR DESCRIPTION
**Pull request by jebikoh**

### CODE REVIEW

**Comment by jebikoh** [comment]
All Changes Present in patch-1

File: spark-kudu examples/scala-kudu/README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ if (kuduContext.tableExists(kuduTableName)) {
 ```
 
 ### **Step 5**: Create the schema
-Enter as many fields as needed. The format is `name, type, nullable?`. Types are SQL types and need to be imported individually. You can find a list of DataTypes [here](https://spark.apache.org/docs/2.0.0/api/java/org/apache/spark/sql/types/DataTypes.html). You can import these by running `import org.apache.spark.sql.types.TYPE`
+Enter as many fields as needed. The format is `name, type, nullable?`. Types are SQL types and need to be imported induvidually. You can find a list of DataTypes [here](https://spark.apache.org/docs/2.0.0/api/java/org/apache/spark/sql/types/DataTypes.html). You can import these by running `import org.apache.spark.sql.types.TYPE`
```
Comment (nw0428) [inline spark-kudu examples/scala-kudu/README.md:79]: individually

File: spark-kudu examples/scala-kudu/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ import org.apache.kudu.client.CreateTableOptions
 
 ### **Step 2:** Set up Kudu masters (1 master)
 
-If there is only 1 master, replace the `#` with the IP, and the % should be the port (usually `7051`. If you have more than one master, look at the alternate steps below Step 3.
+If there is only 1 master, replace the `#` with the IP, and the % should be the port (usually `7051`. If you have more than one master, look at the alternate steps below Step 3.
```
Comment (nw0428) [inline spark-kudu examples/scala-kudu/README.md:33]: Unnecessary space

File: spark-kudu examples/scala-kudu/kudu.scala [hunk 3]
```diff
@@ -77,7 +77,7 @@ val customersRDD = sc.parallelize(customers)
 val customersDF = spark.createDataFrame(customersRDD)
 
 // Step 5
-kuduContext.insertRows(customersDF, kuduTableName) 
+kuduContext.insertRows(df, kuduTableName)
```
Comment (nw0428) [inline spark-kudu examples/scala-kudu/kudu.scala:80]: This seems wrong

**Pull request closed without merging**